.github/workflows/fe-validate-links.yml

```diff
@@ -0,0 +1,50 @@
+name: Frontend Link Validation
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron:  '0 14 * * *'
```

Comment:
So not necessarily tied to a merge but just periodically check?

Reply:
This workflow will just periodically check, but I also added it to the build, via the gradle build file inside the `airbyte-webapp` folder, so that any assemble/build of the gradle project will check it as well. Thus it would be part of our regular Frontend: Build runs. Since some breakage we've seen in the past though came from changes in the docs, I wanted an additional scheduled check.